cli/pbjs.js

```diff
@@ -99,6 +99,8 @@ exports.main = function main(args, callback) {
                 "                   es6       ES6 wrapper (implies --es6)",
                 "                   closure   Just a closure adding to protobuf.roots (see -r)",
                 "",
+                "  --dependency     Specifies which version of protobuf to require. Accepts any valid module id",
+                "",
```

Comment:
If I understood your concern correct, it is actually used in `util.wrap` called by `static-module` and `json-module` targets. This option gets correctly passed down to `(json|static)-module` target.

I only see one concern at the moment: `static-module` target sepcifies default value for `dependency`, but `json-module` - doesn't